docker/Dockerfile

```diff
@@ -61,10 +61,12 @@ RUN git clone https://github.com/AquaticEcoDynamics/libplot.git && \
   cd .. && \
   rm -rf *
 
+# Install some packages which aren't on CRAN or aren't recent enough on CRAN
 RUN Rscript -e 'library(remotes); \
-                install_github("GLEON/GLMr"); \
-                install_github("GLEON/GLM3r"); \
-                install_github("GLEON/glmtools");' \
+                install_github("GLEON/GLMr", upgrade = "never"); \
+                install_github("GLEON/GLM3r", upgrade = "never"); \
+                install_github("GLEON/glmtools", upgrade = "never"); \
+                install_github("USGS-R/ncdfgeom", upgrade = "never");' \
```

Comment:
Don't allow `remotes::install_github()` to update previously installed packages. This is just good Dockerfile practice, won't change functionality here.